src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -13,8 +13,8 @@ export const internalFunctions = {
   listenForNavigationEvents,
   enableNavigationInterception,
   navigateTo,
-  getBaseURI: () => BINDING.js_string_to_mono_string(document.baseURI),
-  getLocationHref: () => BINDING.js_string_to_mono_string(location.href),
+  getBaseURI: () => document.baseURI,
```

Comment:
This file is common to the Server and WebAssembly, so we cannot do this here. Moving this to Boot.WebAssembly. Let me know if there's a better way to do this.

Reply:
Why didn't we use document.baseURI in the past? Doesn't this change any expectation wherever this is used?

Reply:
We did use `document.baseURI` in the past - see the old line 16.